Source/Libraries/openXDA.Model/Emails/EmailType.cs

```diff
@@ -34,21 +34,9 @@ namespace openXDA.Model
     [UseEscapedName]
     public class EmailType : EmailTypeBase
     {
-        [ParentKey(typeof(EmailCategory))]
-        public int EmailCategoryID { get; set; }
-
-        public string TriggerEmailSQL { get; set; }
-
         public string CombineEventsSQL { get; set; }
```

Comment:
May as well fix these too, while we're at it.
```suggestion
        [DefaultValue("SELECT ID FROM Event WHERE ID = {0}")]
        public string CombineEventsSQL { get; set; }
```